services/git-service-api/src/main/java/io/fabric8/launcher/service/git/api/GitUser.java

```diff
@@ -1,14 +1,18 @@
 package io.fabric8.launcher.service.git.api;
 
+import org.immutables.value.Value;
+
 /**
  * Value Object representing a Git user
  *
  * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
  */
+@Value.Immutable
```

Comment:
How much do we gain from this? If it's only about properly made value objects is it really worth an effort to have yet another lombok-inspired annotation processor? Does it have IDE support or do I have to re-run the mvn build to see the changes?

Reply:
It's only about creating the DTOs without requiring us to maintain it. Yes, it does have IDE support, see http://immutables.github.io/apt.html